node/builder.go

```diff
@@ -555,7 +557,7 @@ func ConfigBoost(cfg *config.Boost) Option {
 		Override(new(*dagstore.DAGStore), func() *dagstore.DAGStore { return nil }),
 		Override(new(*mdagstore.Wrapper), func() *mdagstore.Wrapper { return nil }),
 
-		Override(new(pdtypes.Store), modules.NewPieceDirectoryStore(cfg)),
```

Comment:
I don't think we need the `pdtypes.Store` interface at all - we can just use the concrete type. As far as I can tell, the interface to `boostd-data/client` is not adding anything and is redundant.